### ISSUE TITLE
Map view

### ISSUE THREAD

**Issue by Jorieke** [jarmokivekas/hotosm-collate#2]
Hello @jarmokivekas , 
We absolutely love your hotosm collate tool!  
We however were thinking that it would be great to have as well a map view on the page as well.  
This would allow to quickly visually understand what areas that are mapped or not. 
Of course as such not a real issue, but sharing it here with you for inspiration. 
Thanks a lot!

**Comment by jarmokivekas** [comment]
@Jorieke Yes, I absolutely agree there should be a map view as well to display the results from the search. It is something I have had in my plans for some time, but haven't found the time to implement yet.

Especially I would like it to show the task geometry of the projects in the search result, and some of the most important stats.

**Comment by Jorieke** [comment]
@jarmokivekas great, thanks a lot! 
I wanted to share for your inspiration this Umap of the current Gaza activation. One that I find very useful!
https://umap.openstreetmap.fr/en/map/gaza-update_1033966?scaleControl=false&miniMap=false&scrollWheelZoom=false&zoomControl=true&editMode=disabled&moreControl=true&searchControl=null&tilelayersControl=null&embedControl=null&datalayersControl=true&onLoadPanel=undefined&captionBar=false&captionMenus=true#11/31.4342/34.4202

**Comment by jarmokivekas** [comment]
This is getting implemented in pull request #5

**Comment by Jorieke** [comment]
This is absolutely fantastic, thanks @jarmokivekas !

**Comment by jarmokivekas** [comment]
I have made the first revision of this feature, and there are more specific issues on how to continue developing it, so I'll close this as done. But the Map View will get more features over the winter holidays for sure.
